pkg/network/dns_parser.go

```diff
@@ -183,7 +191,10 @@ func (*dnsParser) extractCNAME(domainQueried []byte, records []layers.DNSResourc
 
 func (*dnsParser) extractIPsInto(alias, domainQueried []byte, records []layers.DNSResourceRecord, t *translation) {
 	for _, record := range records {
-		if record.Type != layers.DNSTypeA || record.Class != layers.DNSClassIN {
+		if record.Class != layers.DNSClassIN || !isWantedRecordType(record.Type) {
```

Comment:
I'm wondering if `isWantedRecordType` (which probably should be named `isWantedQueryType`) shouldn't be used here. 

Only `A` and `AAAA` records will have IPs addresses, which are used for the IP->domain mapping. That is a somewhat orthogonal feature from breaking down stats by query type. It is possible to not want A/AAAA queries captured for stats purposes, but still wanting the `domain` tag to be added.